Dockerfile

```diff
@@ -2,11 +2,11 @@ FROM python:3.12-alpine
 
 LABEL maintainer="https://github.com/prowler-cloud/prowler"
 
-# Update system dependencies and install essential tools
-#hadolint ignore=DL3018
-RUN apk --no-cache upgrade && apk --no-cache add curl git
+# Install essential tool to compile presidio-analyze library
+# hadolint ignore=DL3018
+RUN apk --no-cache add g++
```

Comment:
Why are you removing the other packages installation?

Reply:
I think they are not needed, but I can leave them if necessary.